library/X509/Job.php

```diff
@@ -99,9 +99,10 @@ private static function generateTargets(ConfigObject $jobDescription, array $hos
 //            }
             $ipv6 = filter_var($start_ip, FILTER_VALIDATE_IP, FILTER_FLAG_IPV6) !== false;
             $subnet = $ipv6 ? 128 : 32;
-            $ip_count = 1 << ($subnet - $prefix);
+            $ip_count = pow(2, ($subnet - $prefix) - 2);
             $start = static::addrToNumber($start_ip);
-            for ($i = 0; $i < $ip_count; $i++) {
+            $scanLimit = $jobDescription->get('scan_limit', pow(2, 327831396));
```

Comment:
> php > echo pow(2, 327831396);
> INF

:thinking: :rofl:

Reply:
Should actually match this value 🙈 `7.391.088.000`